src/compiler/checker.ts

```diff
@@ -2,6 +2,8 @@
 
 /* @internal */
 namespace ts {
+    const ambientModuleSymbolRegex = /^".+"$/;

```

Comment:
I was copying the behavior where we access the modules elsewhere in the checker (see `resolveExternalModuleNameWorker()`)